### PR TITLE
docs: update static to isr

### LINKED ISSUE #20963: docs: "Rendering Modes" change `static` to `isr` in description list
### Environment

https://nuxt.com/docs/guide/concepts/rendering#hybrid-rendering

### Reproduction

Visit https://nuxt.com/docs/guide/concepts/rendering#hybrid-rendering. 

### Describe the bug

`isr` is used in code sample while `static` is used in the `ul` of descriptions below:
image

### Additional context

_No response_

### Logs

_No response_

### PR DESCRIPTION
**Pull request by nateemerson**
### 🔗 Linked issue

#20963

### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation, readme or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Update key in descriptions list of "Rendering Modes" documentation page to use `isr` and match code snippet.

Resolves #20963

### 📝 Checklist

- [X] I have linked an issue or discussion.
- [X] I have updated the documentation accordingly.

### CODE REVIEW

**Review by danielroe — APPROVED** [review]

File: docs/2.guide/1.concepts/3.rendering.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ The different properties you can use are the following:
 - `cors` - Automatically adds cors headers with `cors: true` - you can customize the output by overriding with `headers`
 - `headers` - Add specific headers to sections of your site - for example, your assets
 - `swr` - Add cache headers to the server response and cache it in the server or reverse proxy for a configurable TTL. The `node-server` preset of Nitro is able to cache the full response. For Netlify and Vercel, the response is also added to the CDN layer.
```
Comment (danielroe) [inline docs/2.guide/1.concepts/3.rendering.md:106]:
```suggestion
- `swr` - Add cache headers to the server response and cache it in the server or reverse proxy for a configurable TTL. The `node-server` preset of Nitro is able to cache the full response.
```

**Comment by nateemerson** [comment]
@danielroe's suggestions incorporated!

**Comment by danielroe** [comment]
Thank you ❤️

**Comment by flexchar** [comment]
Is it correct to assume that isr/swr provides that same behaviour as static? 

I'm having trouble understanding the variety of options. The re-rendering sounds very appealing but the reason for going for Static Site Generation mode is the highest performance. Which seems to be lost when using swr/isr.

**Pull request merged**